### PR TITLE
[App Store] Paywong Payments

### PR DESCRIPTION
**Pull request by frspaywong**
## What does this PR do?

An app to integrate Cal.com and Paywong (Crypto Payment Gateway). Paywong allows users to accept crypto payments upon booking a time.

## Current Issues 
Clicking on confirm booking leads to no credentials error. Would appreciate any help on this

**Environment**: Staging(main branch)

## How should this be tested?
No tests set up for this yet.

## Type of change
- New feature (non-breaking change which adds functionality)

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by frspaywong** [comment]
Could someone point me to what's causing the no credentials error?

**Comment by alannnc** [comment]
> Could someone point me to what's causing the no credentials error in the createBooking fetch function?
> 
> ```
> prisma:query SELECT "public"."Credential"."id" FROM "public"."Credential" WHERE ("public"."Credential"."type" = $1 AND "public"."Credential"."userId" = $2) LIMIT $3 OFFSET $4
> @calcom/web:dev: prisma:error 
> @calcom/web:dev: Invalid `prisma.credential.findFirstOrThrow()` invocation:
> @calcom/web:dev: 
> @calcom/web:dev: 
> @calcom/web:dev: An operation failed because it depends on one or more records that were required but not found. Expected a record, found none.
> @calcom/web:dev: 13:24:01.393 ERROR [api] book:user Booking 4 failed Error when saving booking to db No Credential found 
> ```

Already solved, waiting on more updates on this PR.

File: packages/app-store/paywongpayments/services/createPayment.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+export const createPaywongPayment = async (
+    input: ICreatePaywongPayment,
+) => {
+const endpoint = "https://api.paywong.com/v1/graphql";
+
+
+  const graphQLClient = new GraphQLClient(endpoint, {
+    headers: {
+      authorization: `Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJjbGFpbXMiOnsieC1oYXN1cmEtYWxsb3dlZC1yb2xlcyI6WyJhcHAtcHVibGljIl0sIngtaGFzdXJhLWRlZmF1bHQtcm9sZSI6ImFwcC1wdWJsaWMiLCJ4LWhhc3VyYS1hcHAtaWQiOiJkMzE1YTgxNy00MmRlLTRlMzItOTZmMi1kMWYxNWQzNjJlNmEiLCJ4LWhhc3VyYS1hY2NvdW50LWlkIjoiUzdyRm8wRnpZIiwieC1oYXN1cmEtYXBwLXNhbHQiOiJiXy0ifSwiaWF0IjoxNjc5MDMzNTAyfQ.zl-UHhoM4-_mgEaDi3tIhkVojgkhGJiH5MAtB7CgDVY`,
```
Comment (alannnc) [inline packages/app-store/paywongpayments/services/createPayment.tsx:18]: Use dynamic keys from App.keys model.

**Review by hariombalhara — CHANGES_REQUESTED** [review]
Requesting changes on behalf of @alannnc who has left comments.

**Pull request closed without merging**